json.c

```diff
@@ -804,6 +822,7 @@ static int json_get_number_size(struct json_parse_state_s *state) {
       if (json_parse_flags_allow_equals_in_object & flags_bitset) {
         break;
       }
+      // FALLTHROUGH
```

Comment:
Gurgh - is this to stop a compiler warning?

Reply:
yep. with gcc 8.2.1, when doing `cd test; cmake .; make`
the build fails with 

```json.c: In function 'json_get_number_size':
json.c:799:10: error: this statement may fall through [-Werror=implicit-fallthrough=]
       if (json_parse_flags_allow_equals_in_object & flags_bitset) {
          ^
json.c:802:5: note: here
     default:
     ^~~~~~~
cc1: all warnings being treated as errors
```